libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/Edge.kt

```diff
@@ -92,15 +91,15 @@ data class Edge(
     /**
      * The edge's curvature
      */
-    var curvature: Int,
+    var curvature: Byte,
 
     // Optional properties //
     // Only when requested //
 
     /**
      * Optional geometry if requested.
      */
-    var geometry: LineString? = null,
+    var geometry: Geometry? = null,
```

Comment:
Actually it always will be `LineString`. `Geometry` is only used here, because we cannot use `LineString`, because of restrictions on iOS(we can use either `Geometry` either `[coordinate]` there)

Reply:
LineString or empty if `include_geometries == false`